rpc/qubesbuilder.FileCopyIn

```diff
@@ -67,6 +67,11 @@ def main():
 
     # Move the file to the destination directory
     dn.mkdir(parents=True, exist_ok=True)
+    if (dn / bn).exists():
+        if (dn / bn).is_dir():
+            shutil.rmtree(dn/bn)
+        else:
+            (dn / bn).unlink()
```

Comment:
Is that really right thing to do? I think fresh dispvm has all this /builder empty, so if anything is there, it was just copied in (in another copy_in call?), in which case removing it feels wrong